submissions/VasylRosokha/Popup/index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Popup</title>
+</head>
+<body>
+
+    <header class="header">
```

Comment:
Add a mobile version of your popup.
![image](https://user-images.githubusercontent.com/16196199/190704559-64e64ee6-6782-486b-8390-ce01a5732eed.png)